pypdf/_writer.py

```diff
@@ -1058,7 +1058,7 @@ def update_page_form_field_values(
             raise PyPdfError("No /AcroForm dictionary in PdfWriter Object")
         af = cast(DictionaryObject, self._root_object[CatalogDictionary.ACRO_FORM])
         if InteractiveFormDictEntries.Fields not in af:
-            raise PyPdfError("No /Fields dictionary in Pdf in PdfWriter Object")
+            raise PyPdfError("No /Fields dictionary in PdfWriter Object")
```

Comment:
Could you please elaborate the reason for this change?

Reply:
More similar to line 1058, and the PDF and the PdfWriter Object are equivalent.

Reply:
Yes and no. Strictly speaking, a PdfWriter holds a custom representation of the actual PDF structure. Nevertheless, this change is unrelated to the readability aspects IMHO.